metals/src/main/scala/scala/meta/internal/metals/Compilations.scala

```diff
@@ -115,15 +115,6 @@ final class Compilations(
   def cancel(): Unit = {
     cascadeBatch.cancelAll()
     compileBatch.cancelAll()
-    buildTargets.all
-      .flatMap { target =>
-        buildTargets.buildServerOf(target.getId())
-      }
-      .distinct
-      .foreach { conn =>
-        conn.cancelCompilations()
```

Comment:
I think this was done on purpose since the cancelation was not properly propagated before